Week2/README.md

```diff
@@ -4,305 +4,23 @@
 
 These are the topics for week 2:
 
-1. What’s an identifier (keys)?
-   - Primary key
-   - Foreign key
-   - Composite key
-2. What are the 3 types of relationships?
-   - One-to-One (1-1)
-   - One-to-Many (1-M)
-   - Many-to-Many (M-M)
-3. What are indexes?
-4. What’s domain modeling?
-   - Entity Relationship Diagram (ERD)
-5. How to use SQL: joins, aggregate Functions, distinct, group by, having
-   - Joins
-   - Aggregate functions
-   - Distinct
-   - Group By
-   - Having
+1. [Identifiers (Primary key, Foreign key, Composite key)](https://hackyourfuture.github.io/study/#/databases/sql/identifiers)
```

Comment:
These topics align have the corresponding material in the study material! Nice 🎉